print/assets/print-preview.css

```diff
@@ -17,6 +17,11 @@
     margin: 35px auto;
   }
 
+  .tw-break-after-page.landscape-page {
+    max-width: 1123px; /* page width: A4 landscape width of 297mm at 96dpi */
+    min-height: 794px; /* page height: A4 landscape height of 210mm at 96dpi */
+  }
```

Comment:
What is this for?

Reply:
This is for the developer HTML preview which you can see in the iframe on the print page.